lib/partition_setup.pm

```diff
@@ -408,24 +408,57 @@ sub take_first_disk_storage_ng {
     # On ipmi we always have unformatted drive
```

Comment:
You should delete this comment as well as it is no longer true (and in fact it is a reason for this PR)